src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -221,7 +221,7 @@
           {:type :error
            :size :default
            :icon :i/info}
-          error]
+          (i18n/label :t/wrong-password)]
```

Comment:
it's not always wrong password

Reply:
Should we make that visible to the end user?

Reply:
yes, because why would we want to confuse user with the wrong error? he might think app is hacked or smthg

Reply:
Hm right, we should check error messages either way so to be more user friendly, but for this case let's wait for 
@pavloburykh logs so maybe we can fix it on status-go side. I couldn't reproduce it by myself

Reply:
@briansztamfater @flexsurfer attached logs from nightly build here https://github.com/status-im/status-mobile/issues/16038#issuecomment-1576528847